examples/url-sync-json/src/components/Counter/recoil.ts

```diff
@@ -0,0 +1,8 @@
+import { number } from '@recoiljs/refine'
+import { syncEffect } from 'recoil-sync'
+import { atomFamilyWithInitialValue } from 'recoil-sync-next'
+
+export const counter = atomFamilyWithInitialValue<number>({
+  key: 'counterState',
+  effects: [syncEffect({ storeKey: 'url-json-store', refine: number() })],
+})
```

Comment:
I prefer defining this atom family in the `counter.tsx` because it's not shared with others.

Reply:
I fixed it.